packages/lib-ethers/README.md

```diff
@@ -10,7 +10,7 @@ Install in your project:
 npm install @sovryn-zero/lib-base @sovryn-zero/lib-ethers ethers@^5.0.0
 ```
 
-Connecting to an Ethereum node and sending a transaction:
+Connecting to a blockchain node and sending a transaction:
```

Comment:
this one will do but it is out of my scope - i'm  only fixing contracts/contracts 
this one is typescript SDK folders which is managed by CoL